modules/oe_theme_helper/src/Plugin/Condition/CurrentComponentLibraryCondition.php

```diff
@@ -105,8 +117,9 @@ public function evaluate() {
     if (empty($this->configuration['component_library'])) {
       return TRUE;
     }
+    $theme = $this->themeManager->getActiveTheme()->getName();
 
-    $component_library = $this->configFactory->get('oe_theme.settings')->get('component_library');
+    $component_library = theme_get_setting('component_library', $theme);
```

Comment:
Use the theme name in 
```
     $component_library = $this->configFactory->get($theme . '.settings')->get('component_library');
```
Add a condition to the if above to make sure it's not empty too.